kura/org.eclipse.kura.web2/src/main/java/org/eclipse/kura/web/client/util/FailureHandler.java

```diff
@@ -38,7 +38,9 @@ public static void handle(Throwable caught, String name) {
         if (caught instanceof StatusCodeException) {
             final StatusCodeException statusCodeException = (StatusCodeException) caught;
             if (statusCodeException.getStatusCode() == 401) {
+                showErrorMessage("The session is expired", null);
```

Comment:
Please change error message to "The session has expired"
Is there a way to add a timeout, because for now, this error message is only briefly showed and if user is not paying attention, he might miss what happened.